tests/integration/_lib/sandbox.js

```diff
@@ -34,10 +53,17 @@ async function stop () {
   })
 
   await architectSandbox.end()
-  console.log('sandbox ended')
+  startPromise = null
 }
 
+/** At the end of all tests. */
+test.onFinish(() => {
+  console.log('All tests done.  Shutting down sandbox ...')
+  _stop()
+  console.log('Sandbox shut down.')
+
+})
+
 module.exports = {
-  start,
-  stop
+  start
```

Comment:
Same as above, why exporting in an object?

Reply:
Originally there were other functions, but now that it's only start I'll switch this.  Thanks!

Reply:
I tried changing this to module.exports in various ways, but I'm calling `start()` right after this definition, and with various permutations of this (2 sample tries below), things either failed or felt clumsy.  I'll leave it as-is, but if there's a better way, please let me know!

## Try 1 -- `module.exports = async function start () { ... }`

doesn't work

| tried | error |
| --- | --- |
| `start()` | `ReferenceError: start is not defined` |
| `exports.start()` | `exports.start is not a function` |

## Try 2 -- `module.exports.start = async function () {...}`

This works, but looks clumsy to me:

```
module.exports.start = async function () {...}
exports.start()
```

Reply:
Open to suggestions on this one, perhaps there's a sharp way to do it.

Reply:
I'll take a look